package.json

```diff
@@ -64,6 +64,7 @@
     "graphql": "^16.0.0",
```

Comment:
I think we are not testing this behavior in CI, right? We should. The `peerDependency` might not be correct.

Reply:
I could try adding CI tests that install `graphql@17` if you can point me to the right direction - how and where you add them?

Also, what do you mean by `peerDependency not correct`? It didn't change in my PR, right?

Reply:
How would an end user use defer? As you can see we depend upon `graphql`.

Reply:
I see. So the Apollo Server, as a reference, didn't update graphql version and they will only do that when graphql@17 is released. Until that, it's the change that is implemented and tested on CI, but not available to the end users since it's still in alpha

Reply:
That being said, the idea of mine was to do the same here - start supporting it but not allow users to use it until `graphql@17` is officially released

Reply:
In the future, when `graphql@17` is live, we will update the mercurius dependency and everyone will be able to use defer

Reply:
I'm happy to cut a next branch for you, so we can land this with the alpha version of graphql-js.

wdyt?

Reply:
+1

Reply:
I incorporated the feedback and going to see what's going on there with JIT compiler when I have some time